src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BALL_NUMBER = 100;
```

Comment:
```suggestion
    private static final int MAX_BALL_NUMBER = 100;
```

Reply:
I had a thought. Why in class ColuSupplier I created global variable, but not here. I then I remembered that I just afraid to do it. So this is new experience - not no be afraid to change own code. Thank You for this experience.